configure.ac

```diff
@@ -56,7 +56,7 @@ AC_DEFINE_UNQUOTED([MAX_CONFIGS], [$MAX_CONFIGS], [Maximum number of config file
 case "$host" in
 	*-mingw*)
 		CPPFLAGS="${CPPFLAGS} -DWIN32_LEAN_AND_MEAN"
-		CPPFLAGS="${CPPFLAGS} -D_WIN32_WINNT=NTDDI_WINXP"
+		CPPFLAGS="${CPPFLAGS} -D_WIN32_WINNT=NTDDI_VISTA"
```

Comment:
Yes we should change this, but why are we using NTDDI_xxx  to set _WIN32_WINNT? It should be  _WIN32_WINNT_VISTA, isn't it? The former has 8 bytes while _WIN32_WINNT is 4 bytes. Is there some magic by which either form works?